components/support/rs_client/src/tests/test_http_client.rs

```diff
@@ -0,0 +1,122 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
this is purely stylistic and a personal preference, but this is file and http_client.rs are both small enough that I'd be inclined to include these tests directly in http_client.rs. But feel free to keep it this way if you really do prefer it this way!

Reply:
Putting tests in the same file is definitely new to me, but I'm here for it! This explains why I wasn't seeing test files in the support components.